tests/storage/cephfs/conftest.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+@pytest.fixture(scope='session')
+def host_with_ceph(host):
+    assert not host.file_exists('/usr/sbin/mount.ceph'), \
+        "mount.ceph must not be installed on the host at the beginning of the tests"
+    host.yum_install(['centos-release-ceph-jewel'], enablerepo="base,extras", save_state=True)
+    host.yum_install(['ceph-common'], enablerepo="base,extras")
```

Comment:
Perhaps we should one day allow to install more than 1 package with this method.

Reply:
It's already possible, but that wouldn't work here because the release package adds a new repo so you need a second yum install call to install ceph-common.